### PR TITLE
Fix #1076 Migrate article: Preparing for high load Fix #1076

### LINKED ISSUE #1076: Migrate article: Preparing for high load
Migrate the content from this article to fit the documentation portal format https://help.aiven.io/en/articles/4660337

(Assign this issue to yourself before you start, and remember to mention the issue number when you open your pull request)

### PR DESCRIPTION
**Pull request by sluhur**
Fix #1076

# What changed, and why it matters

### CODE REVIEW

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur 
Thanks for this. The screenshots are showing an old version of our UI, could you please check if:
- They are really neeed
- if yes, you can create an updated version of them
We're trying to minimise screenshots since our UI is always evolving and keeping them up to date can become a huge task.
Furthemore, there's a warning in the build, can you check?

**Comment by sluhur** [comment]
> Hi @sluhur Thanks for this. The screenshots are showing an old version of our UI, could you please check if:
> 
> * They are really neeed
> * if yes, you can create an updated version of them
>   We're trying to minimise screenshots since our UI is always evolving and keeping them up to date can become a huge task.
>   Furthemore, there's a warning in the build, can you check?

I've checked the screenshots and they're not really needed. I've removed them all.

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur, a couple of comments to address:
* Titles: avoid the `ing` and use the present. e.g. `Preparing for high load` -> `Prepare for high load`
* Article position, is this an howto?
* There's a lot of content that is written here (e.g. Plans, upgrades, maintenance, forks) that could be just referenced to other articles with a link
* Check the links, some of the content is already moved to devportal

**Comment by ftisiot** [comment]
Hi @sluhur, thanks for this, please re-check the PR description. I had to change it to add 
```
Fix #issue_number
```
If you don't prefix with `Fix`, merging the PR will **not** close the related issue

File: _toc.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ entries:
       - file: docs/platform/concepts/enhanced-compliance-env
       - file: docs/platform/concepts/disaster-recovery-test-scenarios
       - file: docs/platform/concepts/choosing-timeseries-database
-      - file: docs/platform/concepts/service-level-agreement
```
Comment (ftisiot) [inline _toc.yml:35]: Why service level agreement is gone?

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @sluhur, I believe I'm making the same content as per #1335 

I believe we need to check in depth what makes sense to be ported in devportal and how an article fits the [contributing guide](https://github.com/aiven/devportal/blob/main/CONTRIBUTING.rst)

I believe this is valuable info, but how's it different from https://github.com/aiven/devportal/pull/1335? What type of content should this be? a concept/reference/howto?

Also please take some time to port the format correctly to the devportal style, I currently see:

Notes that are not devportal notes style
Links with the link full text and not a caption
Reference to support which is unnecessary
Doing some of this thinking/work upfront during the porting would mean providing an article which is closer to be merged to devportal and therefore needing less interaction from a review point of view!

**Comment by laysauchoa** [comment]
Hey, I was taking a look in this PR and it seems to have mixed content between HowTo and Concepts. The section is "Prepare for high load" so seems to fit in "HowTo" like "How to Prepare for High Load". However, under this section there are things like Plans etc which are not a HowTo topic. So I'd suggest to go over the topics and place it where they are more appropriate.

**Comment by lornajane** [comment]
@sluhur There's some good work here, could you take a look at the requested changes, or let us know if you aren't available to work on this?

**Comment by ftisiot** [comment]
@laysauchoa @lornajane: I took the great work that @sluhur already did and changed a bit the style by:
- referencing to existing other pages when needed
- adding sections as Tips/Warnings
I believe it's ready for review

**Review by dewan-ahmed — APPROVED** [review]
Thanks for the latest fixes @ftisiot 

LGTM

Thank you.

**Pull request merged**